### PR TITLE
[zh-tw] Probabilities and Statistics

### PR DESCRIPTION
**Pull request by kevingo**
I've finish the Probabilities and Statistics part 👍

### CODE REVIEW

**Comment by shervinea** [comment]
Awesome, thanks for your work @kevingo! Now, let's wait for other native speakers to review the translation.

File: zh-tw/refresher-probability.md [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+21. **Random variable ― A random variable, often noted X, is a function that maps every element in a sample space to a real line.**
+
+&#10230;
+隨機變數 - 一個隨機變數 X，它是一個將樣本空間中的每個元素映射到實數域的函數
+<br>
+
+22. **Cumulative distribution function (CDF) ― The cumulative distribution function F, which is monotonically non-decreasing and is such that limx→−∞F(x)=0 and limx→+∞F(x)=1, is defined as:**
+
+&#10230;
+累積分佈函數 (CDF) - 累積分佈函數 F 是單調且不減的函數，其 limx→−∞F(x)=0 且 limx→+∞F(x)=1，定義如下：
```
Comment (johnnychhsu) [inline zh-tw/refresher-probability.md]: 如果改成單調遞增呢?

Comment (kevingo) [reply zh-tw/refresher-probability.md]: 單調遞增比較簡潔通用，感謝！

**Comment by kevingo** [comment]
@johnnychhsu 已調整，感謝，請再幫忙確認一下還有沒有其他需要調整的部分 👍

**Comment by johnnychhsu** [comment]
我這邊應該沒問題了～辛苦了！

**Comment by kevingo** [comment]
@johnnychhsu 感謝！

**Comment by kevingo** [comment]
Hi @shervinea, we've finished the review process. Thanks!

**Comment by shervinea** [comment]
Awesome, thank you @kevingo @johnnychhsu for your work!

**Pull request merged**